scripts/simex

```diff
@@ -394,6 +395,13 @@ builds_remake_parser.set_defaults(cmd=do_builds_remake)
 # ---------------------------------------------------------------------------------------
 
 def do_develop(args):
+	# Initialize the cache.
```

Comment:
Looking at the size of this function I recommend moving this to a dedicated function instead of adding a comment:
```python
if not os.path.isfile("./simex.cache"):
    init_cache()
```